src/app/functions/server/user.ts

```diff
@@ -5,47 +5,50 @@
 import { Session } from "next-auth";
 import { getBillingAndSubscriptions } from "../../../utils/fxa";
 import { getSubscriberByFxaUid } from "../../../db/tables/subscribers";
+import { logger } from "./logging";
 
 /* c8 ignore start */
-export async function checkUserHasMonthlySubscription(user: Session["user"]) {
-  if (!user.subscriber?.fxa_uid) {
+export type SubscriptionType = "yearly" | "monthly" | "bundle";
+
+export async function getUserSubscriptionType(
```

Comment:
This used to return a boolean but may now return `undefined`, will callers handle that OK? Does that just mean the user has no subscription? I think that's fine if so, if you want to be explicit you could have a `SubscriptionType` of `none` but I don't think that's critical (it's pretty clear that `undefined` would mean no subscription IMHO)